### ISSUE TITLE
SafeExamBrowser.msi cannot be found

### ISSUE THREAD

**Issue by yongga-radit** [SafeExamBrowser/seb-win-refactoring#678]
**IMPORTANT**
Please _always_ consult the documentation first before creating a bug report!
https://safeexambrowser.org/windows/win_usermanual_en.html

**Describe the Bug**
Re-installing SEB into 3.3 version

**Steps to Reproduce**
1. Go to Add or Remove Program
2. Click on latest Safe Exam Browser and uninstall
3. While uninstalling, pop up "SafeExamBrowser.msi" is not exist on C:/ProgramData/Downloaded Installation/{10384xxxxx} (maybe already I cleaned up unaccidentally)

**Expected Behavior**
Latest Version Uninstalled and I can Install 3.3 version

**Screenshots**
![03868e5e-9df5-4dd6-9955-23dc2f4b8077](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/112047095/9dde6f72-4e00-4d31-b208-3993591eb377)

**Version Information**
 - OS: [e.g. Windows 11 Home Single Language, Version 2H22]
 - SEB-Version [SEB 3.3.2]

**Additional Context**

**Comment by dbuechel** [comment]
Yes, you most likely deleted the original installer by accident. Furthermore, are you using the official SEB or a product from another vendor? The installers for SEB 3.x all contain the version number and are thus differently named from what is shown in the image, unless you still had SEB 2.x installed and are now trying to upgrade to SEB 3.x? If so, then you'd need to fix the issue manually (you should find instructions on how to do so online) by using the installer of SEB 2.x which still can be found on our SourceForge site.

**Comment by yongga-radit** [comment]
the latest version was on August 2022
